plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -202,11 +208,13 @@ void MeshImportContext::gatherFaceConnectsAndVertices()
   // According to the docs for UsdGeomMesh: If 'normals' and 'primvars:normals' are both specified, the latter has precedence.
   const TfToken primvarNormalsToken("primvars:normals");
   TfToken interpolation = mesh.GetNormalsInterpolation();
+  bool isIndexed = false;
   bool hasNormalsOpinion = false;
   if(mesh.HasPrimvar(primvarNormalsToken))
   {
     UsdGeomPrimvar primvar = mesh.GetPrimvar(primvarNormalsToken);
     interpolation = primvar.GetInterpolation();
+    isIndexed = primvar.IsIndexed();
```

Comment:
Nobody is using isIndexed later? 

error: variable ‘isIndexed’ set but not used [-Werror=unused-but-set-variable]

Reply:
This PR predates us updating our build to use the strict compilation mode - will do a pass through it... thanks